src/vs/workbench/browser/parts/editor/multiRowEditorTabsControl.ts

```diff
@@ -64,7 +64,8 @@ export class MultiRowEditorControl extends Disposable implements IEditorTabsCont
 		const [editorTabController, otherTabController] = this.model.isSticky(editor) ? [this.stickyEditorTabsControl, this.unstickyEditorTabsControl] : [this.unstickyEditorTabsControl, this.stickyEditorTabsControl];
 		const didChange = editorTabController.openEditor(editor, options);
 		if (didChange) {
-			// HACK: To render all editor tabs on startup, otherwise only one row gets rendered
+			// We have to rerender both tab bars as the previous active tab could have moved between the two
+			// and we need to ensure that the active tab indicators have been removed from the previous active tab
```

Comment:
What I find misleading here is that we call `openEditors` with an empty `[]` of editors, indicating to the reader that all editors of that other tab bar are now gone?

If we need a way to redraw the other tab row, I would make it an explicit method for this purpose that does not open an empty array of editors.

Reply:
It does not mean that all editors are gone, it means that no new editors have been newly opened. But yes, this is still not nice. Maybe we can use `updateEditorLabel` as this what is mainly needed.

Reply:
✅